object_detection/scripts/get_objects_and_coordinates.py

```diff
@@ -0,0 +1,136 @@
+"""
+This script test the trained model and performs object detection on a given image, the script prints
+in console a JSON with the objects detected and their info.
+
+E.g. 
+>>> python get_objects_and_coordinates.py -d test_images/image1.jpg -i false
+output:
+{
+    "jumex_cajita": {
+        "score": 0.9995954632759094,
+        "ymin": 435.3594779968262,
+        "xmin": 185.599547624588,
+        "ymax": 547.5826263427734,
+        "xmax": 248.1843888759613
+    },
+    "fanta": {
+        "score": 0.9212740063667297,
+        "ymin": 314.3073320388794,
+        "xmin": 328.5956025123596,
+        "ymax": 473.17280769348145,
+        "xmax": 405.4909944534302
+    }
+}
+"""
+
+import os
+import argparse
+import cv2
+import numpy as np
+import tensorflow as tf
+import sys
+import json
+
+from utils import label_map_util
+from utils import visualization_utils as vis_util
+
+from google.colab.patches import cv2_imshow
```

Comment:
I think this might break if not run in colab. Maybe use the normal one?

Reply:
Yes it will, this is not the final script that will be used on the jetson. I will update this as soon as I start working with the jetson, thx.